packages/image_picker/image_picker_android/pubspec.yaml

```diff
@@ -26,3 +26,9 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   mockito: ^5.0.0
+
+# FOR TESTING ONLY. DO NOT MERGE.
+dependency_overrides:
+  image_picker_platform_interface:
+    path:
+      ../image_picker_platform_interface
```

Comment:
add a line to the end of file

Reply:
@cyanglaz Path-based dependency overrides are always temporary lines; this whole block will be reverted before anything can land, in step 4 of [the instructions](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changing-federated-plugins). Given that, we shouldn't worry about whitespace on those bits.